docs/RouterTutorial/react-navigation-tab-kr.md

```diff
@@ -1,414 +1,677 @@
+
+
+
+###### createBottomTabNavigator
+
+createBottomTabNavigator을 사용한 예제에서는 activetintcolor와 inactivetintcolor을 사용해 라벨과 아이콘의 활성화 여부에 따른 색깔을 설정해주었고 style에서 backgroundcolor을 사용해 tab bar의 배경색을 지정해주었습니다. 
+
+<img src="./Tab_createBottomTabNavigator.gif" style="zoom:67%;" />
+
+*CreateBottomTabNavigator.js*
+
+```jsx
+import React from 'react';
+import {createAppContainer} from 'react-navigation';
+import {createBottomTabNavigator} from 'react-navigation-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import HomeScreen from './screens/HomeScreen';
+import ChatScreen from './screens/ChatScreen';
+import SettingsScreen from './screens/SettingsScreen';
+import IconWithBadge from './IconWithBadge';
+
+const HomeIconWithBadge = props => {
+  return <IconWithBadge {...props} badgeCount={3} />;
+};
+
+const TabNavigator = createBottomTabNavigator(
+  {
+    Home: {
+      screen: HomeScreen,
+    },
+    Chat: {
+      screen: ChatScreen,
+    },
+    Settings: {
+      screen: SettingsScreen,
+    },
+  },
+  {
+    defaultNavigationOptions: ({navigation}) => ({
+      tabBarIcon: ({horizontal, tintColor}) => {
+        const {routeName} = navigation.state;
+        let IconComponent = Ionicons;
+        let iconName;
+        if (routeName === 'Home') {
+          iconName = 'ios-home';
+        } else if (routeName === 'Chat') {
+          iconName = 'ios-chatboxes';
+          IconComponent = HomeIconWithBadge;
+        } else if (routeName === 'Settings') {
+          iconName = 'ios-settings';
+        }
+
+        return (
+          <IconComponent
+            name={iconName}
+            size={horizontal ? 20 : 25}
+            color={tintColor}
+          />
+        );
+      },
+    }),
+    tabBarOptions: {
+      activeTintColor: 'white',
+      inactiveTintColor: 'gray',
+      style: {
+        backgroundColor: 'black',
+      },
+    },
+  },
+);
+
+export default createAppContainer(TabNavigator);
+
+```
+
+- `initialRouteName` - The routeName for the initial tab route when first loading.
+
+  첫 로딩시 초기 탭 라우터 이름 설정 
+
+- `defaultNavigationOptions` - Default navigation options to use for screens
+
+  네비게이터에 있는 모든 routes에 디폴트 옵션으로 적용
+
+- ```
+  tabBarOptions
+  ```
+
+  - `activeTintColor` - 활성화된 탭의 라벨과 아이콘 색깔 설정
+  - `activeBackgroundColor` - 활성화된 탭의 배경 색깔 설정
+  - `inactiveTintColor` - 비활성화된 탭의 라벨과 아이콘 색깔 설정
+  - `inactiveBackgroundColor` - 비활성화된 탭의 배경 색깔 설정
+  - `showLabel` - 라벨 show 여부 설정
+  - `showIcon` - 아이콘 show 여부 설정
+  - `style` - tab bar 스타일 설정 
+  - `labelStyle` - label 스타일 설정
+  - `labelPosition` - label 위치 설정
+  - `tabStyle` - 탭 스타일 설정
+  - `adaptive` - 스크린에 따른 아이콘 및 라벨 조정여부 설정
+  - `keyboardHidesTabBar` - 키보드 사용시 탭바 숨김 여부 설정
+
+  
+
+###### createMaterialBottomTabNavigator
+
+createMaterialBottomTabNavigator을 사용한 예제에서는 아래에서 볼 수 있듯이 shifting을 이용해 해당하는 탭의 라벨만 표시되고 activetintcolor, inactivetintcolor, barstyle을 이용해 배경색 역시 각각 활성화 여부에 따라 다르게 지정해주었습니다.
```

Comment:
`activetintcolor` -> `activeTintColor`
`inactivetintcolor` -> `inactiveTintColor`
`barstyle` -> `barStyle`
(for consistency)